ui/src/components/inputs/InputsForm.vue

```diff
@@ -61,14 +75,12 @@
             <el-input
                 type="password"
                 v-if="input.type === 'SECRET'"
-                :data-test-id="`input-form-${input.id}`"
```

Comment:
Why this deletion ? Unrelated I think

Reply:
i believe this was not in the original code, so i will keep it deleted, not required actually

Reply:
After verification, this is in the original code
Please only modify what's needed

Reply:
please check it once again, i have updated, now all the test cases are running.